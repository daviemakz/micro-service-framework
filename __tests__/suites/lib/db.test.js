'use strict';

// Import system components
import Db from '../../../dist/lib/db';

// Test suite
describe('dist/lib/db', () => {
  test('to match snapshot', () => {
    expect(new Db({ databaseName: 'exampleTable' })).toMatchSnapshot();
  });
});
