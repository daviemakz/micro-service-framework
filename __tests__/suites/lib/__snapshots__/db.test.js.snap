// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dist/lib/db to match snapshot 1`] = `
FrameworkDatabase {
  "db": table {
    "add": [Function],
    "all": [Function],
    "delete": [Function],
    "fetch": [Function],
    "fetchAll": [Function],
    "get": [Function],
    "has": [Function],
    "includes": [Function],
    "push": [Function],
    "set": [Function],
    "startsWith": [Function],
    "subtract": [Function],
    "tableName": "exampleTable",
  },
}
`;
